helpers_test.go

```diff
@@ -339,46 +313,46 @@ func assertMessageMatchesEvent(t *testing.T, msgEnv Message, msg *Event) {
 		if event.StreamCategory != msg.StreamCategory {
 			t.Error("StreamCategory in message does not match")
 		}
-		data := new(dummyData)
-		err := Unpack(event.Data, data)
-		if err != nil {
-			t.Error("Couldn't unpack data from message")
+		if !reflect.DeepEqual(msg.Data, event.Data) {
+			t.Error("Messages data are not correct")
 		}
-		if !reflect.DeepEqual(&dummyData{"a"}, data) {
-			t.Error("Messages are not correct")
+		if !reflect.DeepEqual(msg.Metadata, event.Metadata) {
+			t.Error("Messages metadata are not correct")
 		}
 	default:
-		t.Error("Unknown type of Message")
+		t.Errorf("Unknown type of Message message type: %T", event)
 	}
 }
 
-func assertMessageMatchesOtherMessage(t *testing.T, msgEnv Message, msg *otherMessage) {
```

Comment:
if this helper is no longer used, remove it (it looks like it was used in the Test that was commented out)